discord/interactions.py

```diff
@@ -938,13 +962,18 @@ async def send_modal(self, modal: Modal) -> Interaction:
         if self._responded:
             raise InteractionResponded(self._parent)
 
+        parent = self._parent
+
         payload = modal.to_dict()
         adapter = async_context.get()
+        http = parent._state.http
         await self._locked_response(
             adapter.create_interaction_response(
                 self._parent.id,
                 self._parent.token,
                 session=self._parent._session,
```

Comment:
Since `parent` is now defined
```suggestion
                parent.id,
                parent.token,
                session=parent._session,
```